frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -95,7 +95,7 @@ export class Gallery extends React.Component {
 
   liveMessageNotRelevant (data, state) {
     return Number(data.content.workspace_id) !== Number(state.config.appConfig.workspaceId) ||
-    Number(data.content.parent_id) !== Number(state.folderId)
+    (state.folderId !== undefined && Number(data.content.parent_id) !== Number(state.folderId))
```

Comment:
The TLM is relevant if `Number(data.content.parent_id) !== Number(state.folderId)` ?

Reply:
The TLM is revelent if `Number(data.content.parent_id) === Number(state.folderId)`*

Reply:
ok, my bad, I forgot the `Not` in the function name. PR approuved.